src/Client.php

```diff
@@ -10,9 +10,7 @@ class Client extends GuzzleClient
     public function __construct(Config $config)
     {
         parent::__construct(
-            [
```

Comment:
I think we may need to do a bit of merging of the config here as by removing the `unleash.url` we are not making this backwards compatible.

I am not sure this feature warrants an entire new major version, so I'm just wondering if there's anything clever we could do to fall back to `unleash.url` if `unleash.requestDefaults` doesn't exist? Does this make sense?

Reply:
Oh yes, good point.

I have been hit _twice_ this week by unheralded breaking changes, so I'm very sympathetic to that!

Updated to return `unleash.url` to its original spot.